blueprints/screening_app/__init__.py

```diff
@@ -0,0 +1,420 @@
+from mindmeld import Application
+import screening_app.prediabetes as pd
+
+app = Application(__name__)
+
+
+@app.handle(intent='greet')
+def welcome(request, responder):
+    """
+    When the user begins the conversation with a greeting. Explain the system options.
+    """
+    responder.reply(pd.WELCOME_MSG)
+
+
+@app.handle(intent='exit')
+def say_goodbye(request, responder):
+    """
+    When the user ends a conversation, clear the dialogue frame and say goodbye.
+    """
+    # Clear the dialogue frame to start afresh for the next user request.
+    responder.frame = {}
+
+    # Respond with a random selection from one of the canned "goodbye" responses.
+    responder.reply(['Gracias por su visita. ¡Adios!',
+                     'Gracias por su visita. ¡Hasta luego!',
+                     'Gracias por su visita. Que tenga buen día.'])
+
+
+@app.handle(intent='answer_no')
+def handle_rejection(request, responder):
+    """
+    When the user rejects the screening, gracefully exit the conversation.
+    """
+    # Clear the dialogue frame to start afresh for the next user request.
+    responder.frame = {}
+
+    responder.reply(['Estamos para servirle. Gracias por su visita.'])
+
+
+@app.dialogue_flow(domain='prediabetes_screening', intent='opt_in')
+def screen_prediabetes(request, responder):
+    """
+    If the user accepts the sceening, begin a dialogue flow.
+    """
+    # Clear the dialogue frame to start fresh
+    responder.frame = {}
+
+    first_question = app.question_answerer.get(index='prediabetes_screening_questions',
+                                               question_number=1)
+
+    responder.frame['previous_question'] = first_question[0]['text']
+    responder.frame['previous_question_number'] = first_question[0]['question_number']
+    responder.frame['screening'] = dict()
+
+    responder.reply(first_question[0]['text'])
+    responder.listen()
```

Comment:
Because of the multiple questions within this flow that could get a simple number answer, using allowed_intents help keep the dialogue on track. For example, here if you expect responses to be `answer_age`, but could also be either `exit`, just set
```
responder.params.allowed_intents = ('prediabetes_screening.answer_age', '*.exit')
```
For the yes-no question prompts, it will look more like 
```
responder.params.allowed_intents = (
    'prediabetes_screening.answer_yes',
    'prediabetes_screening.answer_no',
    'prediabetes_screening.answer_yes_active',
    '*.exit'
)
```